### PR TITLE
Bug 1298619, Added SELinux guidance for installation

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1298619

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape , @bfallonf, can you please eyeball this?

**Comment by ahardin-rh** [comment]
Approved by reporter, Eric Jones

**Comment by bfallonf** [comment]
@ahardin-rh Heyo. So this flagged a memory I had from a security talk they gave in Brisbane one day, so I brought it up with one of the security guys (Wade Mealing), and yes, we're not supposed to be recommending using "setenforce=0" anywhere. So just taking out the final part ("Alternatively...") and this should be ok.

My own question is, why would Selinux be disabled? Obviously it's not by default, as I've never had this issue anytime I've tried to install. Also, should we really be recommending that we disable Selinux? From what I can tell, that might be a bad idea security-wise, right?

**Comment by trevorjay** [comment]
@bfallonf You'd be happy to know Wade is who told me about this thread. <sidious>The circle is complete.</sidious>

I would strongly recommend striking how to disable SELinux full stop. It isn't just a security issue (which, of course it is) or a support issue for the community/Red Hat (again, which it is). Disabling SELinux is likely to cause the worst kind of problems with OpenShift. By worst I mean intermittent or seemingly random and that depend on *exactly* what containers you are running. You know, the kind where debugging them destroys your sanity.

**Comment by bfallonf** [comment]
@trevorjay Cool. Thanks for that. 

@ahardin-rh So maybe a warning box about "The OpenShift installer needs Selinux enabled. You shouldn't disable Selinux, but if you do, here's how to enable it again:...". But I'll let you figure out the details :)

**Comment by ahardin-rh** [comment]
@trevorjay @bfallonf thanks for your comments. I removed the details on disabling and kept the focus on the fact that SELinux should be enabled.

**Comment by ahardin-rh** [comment]
**revhistory note:**

For Install Config --> Prerequisites

    Added a link:../install_config/install/prerequisites.htmll#prereq-selinux[SELinux] section to include guidance that SELinux must be enabled, or the installer will fail.

**Comment by trevorjay** [comment]
@ahardin-rh Looks good to me.

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -340,6 +340,28 @@ link:../../install_config/persistent_storage/persistent_storage_gce.html[GCE
 Persistent Disks], and
 link:../../install_config/persistent_storage/persistent_storage_iscsi.html[iSCSI].
 
+[[prereq-selinux]]
+
+=== SELinux
+
+Security-Enhanced Linux (SELinux) must be enabled on all of the servers before
+installing OpenShift or the installer will fail. Also, configure
+`*SELINUXTYPE=targeted*` in the *_etc/selinux/config_* file:
```
Comment (adellape) [inline install_config/install/prerequisites.adoc]: @ahardin-rh Missing a `/` at the beginning of the file path.

Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: Follow-up PR: https://github.com/openshift/openshift-docs/pull/1644

**Pull request merged**